executor/cook/config.py

```diff
@@ -49,12 +52,24 @@ def initialize_config(environment):
     """Initializes the config using the environment.
     Populates the default values for missing environment variables.
     """
+    executor_id = environment.get('MESOS_EXECUTOR_ID', 'executor')
+    sandbox_directory = environment.get('MESOS_SANDBOX', '')
+    default_progress_output_name = '{}.progress'.format(executor_id)
+    if sandbox_directory:
+        default_progress_output_file = os.path.join(sandbox_directory, default_progress_output_name)
+    else:
+        default_progress_output_file = default_progress_output_name
+
+    progress_output_env_variable = environment.get('EXECUTOR_PROGRESS_OUTPUT_FILE_ENV', 'EXECUTOR_PROGRESS_OUTPUT_FILE')
+    logging.info('Progress location environment variable is {}'.format(progress_output_env_variable))
+    if progress_output_env_variable not in environment:
+        logging.warning('No entry found for {} in the environment'.format(progress_output_env_variable))
```

Comment:
Do we need to warn if this isn't set?  Most, if not all, of the time the user won't be setting this.

Reply:
Changed it to `info`.